### PR TITLE
[11.x] Add PHPDoc type hints

### PR DESCRIPTION
**Pull request by shaedrich**
see #53961

### CODE REVIEW

File: src/Illuminate/Support/Facades/Schedule.php [hunk 1]
```diff
@@ -73,13 +73,13 @@
  * @method static \Illuminate\Console\Scheduling\PendingEventAttributes weekly()
  * @method static \Illuminate\Console\Scheduling\PendingEventAttributes weeklyOn(array|mixed $dayOfWeek, string $time = '0:0')
  * @method static \Illuminate\Console\Scheduling\PendingEventAttributes monthly()
- * @method static \Illuminate\Console\Scheduling\PendingEventAttributes monthlyOn(int $dayOfMonth = 1, string $time = '0:0')
- * @method static \Illuminate\Console\Scheduling\PendingEventAttributes twiceMonthly(int $first = 1, int $second = 16, string $time = '0:0')
+ * @method static \Illuminate\Console\Scheduling\PendingEventAttributes monthlyOn(int<0, 31> $dayOfMonth = 1, string $time = '0:0')
+ * @method static \Illuminate\Console\Scheduling\PendingEventAttributes twiceMonthly(int<0, 31> $first = 1, int<0, 31> $second = 16, string $time = '0:0')
```
Comment (timacdonald) [inline src/Illuminate/Support/Facades/Schedule.php:77]: If this is a zero based month index, should the range be `1-31`? Or does `0` do something special here?

Comment (shaedrich) [reply src/Illuminate/Support/Facades/Schedule.php:77]: Damn, you are right 👍🏻 My bad 🤦🏻 😅

Comment (timacdonald) [reply src/Illuminate/Support/Facades/Schedule.php:77]: I'll send a fix.

Comment (timacdonald) [reply src/Illuminate/Support/Facades/Schedule.php:77]: https://github.com/laravel/framework/pull/53985

**Pull request merged**